### PR TITLE
arrays tasks are completed

### PR DESCRIPTION
**Pull request by emakhachek**

### CODE REVIEW

File: Exercises/arrays/arrays.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Task #1
+function randomArrayGenerator() {
+    let arr = Array.from({ length: parseInt(Math.random() * 100) }, () => Math.floor(Math.random() * 100));
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: return this array for func

File: Exercises/arrays/arrays.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    console.log(arr)
+}
+randomArrayGenerator();
+
+// Task #2
+const array = [1, 2, 3, 4];
+function copyOfArray(ar) {
+    const result = ar.slice();
+    return console.log(result);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: return array from function, and think about an easier copy of array

File: Exercises/arrays/arrays.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+copyOfArray(array);
+
+// Task #3
+const array = [1, 2, 3, 4];
+function findElement(ar, element) {
+    const result = ar.indexOf(element);
+    return console.log(result);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: return result of indexOf

File: Exercises/arrays/arrays.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+findElement(array, 3);
+
+// Task #4
+const array1 = [1, 2, 3, 4];
+const array2 = [4, 5];
+function concatArray(ar1, ar2) {
+    const result = ar1.concat(ar2);
+    return console.log(result);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: return result from func

**Review by lizarusi — CHANGES_REQUESTED** [review]

**Review by Mike-Savin — APPROVED** [review]
👍

**Pull request merged**